### PR TITLE
DOC-1578: Fixed code samples/snippets and the use of pronouns in the Initial configuration section

### PR DESCRIPTION
**Pull request by lnewson**
Related Ticket: DOC-1578

Description of Changes:
- Fixed code samples/snippets in the Initial configuration section
- Fixed the use of pronouns in the Initial configuration section

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: modules/ROOT/partials/configuration/cache_suffix.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [[cache_suffix]]
 == `+cache_suffix+`
 
-This option lets you add a custom cache buster URL part at the end of each request {productname} makes to load CSS, scripts, etc. Just add the query string part you want to append to each URL request, for example "?v=4.1.6".
+This option allows a custom cache buster URL part to be added at the end of each request {productname} makes to load CSS, scripts, etc. Just add the query string part that should be append to each URL request, for example "?v=4.1.6".
```
Comment (tiny-ben-tran) [inline modules/ROOT/partials/configuration/cache_suffix.adoc:4]: append => appended?

File: modules/ROOT/partials/configuration/resize.adoc [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [[resize]]
 == `+resize+`
 
-This option gives you the ability to disable the resize handle or set it to resize both horizontal and vertically. The option can be true, false or the string `+'both'+`. `+False+` disables the resize, `+true+` enables vertical resizing only, `+'both'+` makes it possible to resize in both directions horizontal and vertical.
+This option allows the resize handle to be disabled or to set the resize handler to resize both horizontally and vertically. The option can be `+true+`, `+false+` or the string `+'both'+`. `+False+` disables resizing, `+true+` enables vertical resizing only, `+'both'+` makes it possible to resize in both directions horizontal and vertical.
```
Comment (tiny-ben-tran) [inline modules/ROOT/partials/configuration/resize.adoc:4]: I think dot points are better for listing multi-values.

**Review by tiny-ben-tran — COMMENTED** [review]
Did a quick review. Everything looks good. Not sure about `a user` instead of `an user` in a few places.

File: modules/ROOT/partials/configuration/resize.adoc (hunk 2 above)
Comment (lnewson) [reply modules/ROOT/partials/configuration/resize.adoc:4]: That might be better, but that's well outside the scope of this task so we're better off leaving that for now (even the pronouns tbh were outside of the scope but they are worse).

File: modules/ROOT/partials/configuration/cache_suffix.adoc (hunk 1 above)
Comment (lnewson) [reply modules/ROOT/partials/configuration/cache_suffix.adoc:4]:
```suggestion
This option allows a custom cache buster URL part to be added at the end of each request {productname} makes to load CSS, scripts, etc. Just add the query string part that should be appended to each URL request, for example "?v=4.1.6".
```

**Comment by lnewson** [comment]
> Not sure about `a user` instead of `an user` in a few places.

This isn't something that was changed as part of this, so is out of scope. In saying that though, `a user` is correct: https://english.stackexchange.com/a/105117

File: modules/ROOT/partials/configuration/color_picker_callback.adoc [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 This option enables you to provide your own color picker.
 
-Type : `+JavaScript Function+`
+Type : `+Function+`
 
 === Example: Using `+color_picker_callback+`
```
Comment (EkimChau) [inline modules/ROOT/partials/configuration/color_picker_callback.adoc:8]: we didn't convert this one to a promise?

**Review by EkimChau — APPROVED** [review]

File: modules/ROOT/partials/configuration/color_picker_callback.adoc (hunk 3 above)
Comment (lnewson) [reply modules/ROOT/partials/configuration/color_picker_callback.adoc:8]: It's not in the `Initial configuration` section... this only got changed as I did a Find and Replace on `JavaScript Function` in the type.

That said, `color_picker_callback` doesn't even exist anymore and this file isn't included anywhere in the docs, so we should delete this file.

Comment (lnewson) [reply modules/ROOT/partials/configuration/color_picker_callback.adoc]: Deleted the unused page.

**Review by thedrjim — APPROVED** [review]

**Review by ltrouton — APPROVED** [review]

**Pull request merged**